### PR TITLE
Package ocsigen-toolkit.4.1.0

### PR DESCRIPTION
**Pull request by balat**
### `ocsigen-toolkit.4.1.0`
Reusable UI components for Eliom applications (client only, or client-server)
The Ocsigen Toolkit is a set of user interface widgets that facilitate the development of Eliom applications.

---
* Homepage: http://www.ocsigen.org
* Source repo: git+https://github.com/ocsigen/ocsigen-toolkit.git
* Bug tracker: https://github.com/ocsigen/ocsigen-toolkit/issues/

---
:camel: Pull-request generated by opam-publish v2.3.0

### CODE REVIEW

**Comment by mseri** [comment]
On 32 bits there are lots of failures of the type
```
# File "_none_", line 1:
# Error: Cannot load js_of_ocaml-ppx_deriving_json: this object file uses
#        unsafe features
# make: *** [Makefile:185: _deps/src/widgets/ot_calendar.eliom.server] Error 2
# File "_none_", line 1:
# Error: Cannot load js_of_ocaml-ppx_deriving_json: this object file uses
#        unsafe features
# make: *** [Makefile:185: _deps/src/widgets/ot_carousel.eliom.server] Error 2
```
Should we make them incompatible?

File: packages/ocsigen-toolkit/ocsigen-toolkit.4.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+synopsis: "Reusable UI components for Eliom applications (client only, or client-server)"
+description: "The Ocsigen Toolkit is a set of user interface widgets that facilitate the development of Eliom applications."
+authors: "dev@ocsigen.org"
+homepage: "http://www.ocsigen.org"
+bug-reports: "https://github.com/ocsigen/ocsigen-toolkit/issues/"
+dev-repo: "git+https://github.com/ocsigen/ocsigen-toolkit.git"
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+build: [ make "-j%{jobs}%" ]
+install: [ make "install" ]
+depends: [
```
Comment (balat) [inline packages/ocsigen-toolkit/ocsigen-toolkit.4.1.0/opam:12]:
```suggestion
avalaible: arch != "x86_32" & arch != "arm32"
depends: [
```

**Comment by shonfeder** [comment]
The only CI failures appear to be unrelated at this point: solver timeouts on old opam versions and a windows failure of a dependency.

Thank you for publishing the updates!

**Comment by balat** [comment]
Thank you!

**Pull request merged**